src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -86,7 +86,8 @@
 
 <ng-template #osdUsageTpl
              let-row="row">
-  <cd-usage-bar [total]="row.stats.stat_bytes"
+  <cd-usage-bar [title]="'osd ' + row.osd"
```

Comment:
This is another place where you could/should translate the text, since there is some static text (`osd `) to work with.

In case some text attributes needs a clarification for i18n purposes, [Angular allows you to add some metadata to that](https://angular.io/guide/i18n-common-prepare#i18n-metadata-for-translation):
```suggestion
	 <cd-usage-bar [title]="'osd ' + row.osd" i18n-title="Title of the usage bar of osd  whose id is 'row.osd' "
```
That's just an example, not a recomendation for this case ;-)

Reply:
Thanks, this section could use some more information